docs/content/users/install/performance.md

```diff
@@ -10,6 +10,8 @@ On macOS and Windows with Docker Desktop, allocated resources and mounted filesy
 
 Mutagen can offer a big performance boost on macOS and Windows. It’s fast and doesn’t need any setup; you only need to enable it. Before Mutagen, Mac and Windows users configured NFS for speed improvements—though it requires setup and isn’t as fast.
 
+Mutagen is enabled by default on Mac and Windows but can be disabled globally or also at project level.
```

Comment:
Seems like an “and” situation, and I’d reverse the order to subtly suggest handling per-project since there’s presumably good reason for it being globally enabled by default:

```suggestion
Mutagen is enabled by default on Mac and Windows, and it can be disabled per-project or globally.
```